spec/student_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe Student do
+  before :each do
+    @student = Student.new(17, 'Rose', false)
+  end
+  it 'Should return a Student object' do
+    expect(@student).to be_an_instance_of Student
+  end
+
+  describe '#age' do
+    it 'Should return student age' do
+      expect(@student.age).to eql 17
+    end
+  end
+
+  describe '#name' do
+    it 'Should return student name' do
+      expect(@student.name).to eql 'Rose'
+    end
+  end
+  describe '#parent permission' do
+    it 'Should return parent permission' do
+      expect(@student.parent_permission).to be false
+    end
+  end
```

Comment:
- [ ] It would be better to test the `play_hooky` method as well, kindly add it 😁

Reply:
@cynthiainga Thank you for your insightful review 🌹 
I implemented the required changes 👍